### PR TITLE
orka-desktop 3.0.0-beta.1 (new cask)

### PR DESCRIPTION
**Pull request by cameronbroe**
**Important:** *Do not tick a checkbox if you haven’t performed its action.* Honesty is indispensable for a smooth review process.

_In the following questions `<cask>` is the token of the cask you're submitting._

After making any changes to a cask, existing or new, verify:

- [x] The submission is for [a stable version](https://docs.brew.sh/Acceptable-Casks#stable-versions) or [documented exception](https://docs.brew.sh/Acceptable-Casks#but-there-is-no-stable-version).
- [x] `brew audit --cask --online <cask>` is error-free.
- [x] `brew style --fix <cask>` reports no offenses.

Additionally, **if adding a new cask**:

- [x] Named the cask according to the [token reference](https://docs.brew.sh/Cask-Cookbook#token-reference).
- [x] Checked the cask was not [already refused](https://github.com/search?q=repo%3AHomebrew%2Fhomebrew-cask+is%3Aclosed+is%3Aunmerged+&type=pullrequests) (add your cask's name to the end of the search field).
- [ ] `brew audit --cask --new <cask>` worked successfully.
- [x] `HOMEBREW_NO_INSTALL_FROM_API=1 brew install --cask <cask>` worked successfully.
- [x] `brew uninstall --cask <cask>` worked successfully.

---

New cask audit fails notoriety check as this is a relatively new product from [MacStadium](https://www.macstadium.com/). We already have a few other Orka related tools distributed as Casks though.

### CODE REVIEW

**Comment by p-linnane** [comment]
Hi @cameronbroe. Thanks for the PR. Do you plan on having additional beta releases in the future? If not, we can just make this a stable cask, and update it with stable versions moving forward.

**Comment by cameronbroe** [comment]
Hello, @p-linnane. After some discussion with the team, we're not entirely sure how much of a need there is going to be for additional beta releases other than between major versions updates. We do think it may just be best to go on and make this a stable cask though short-term while we consider how we want to handle pre-release versions going forward. Does this make sense to you as well?

File: Casks/o/orka-desktop.rb [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+  sha256 "73979664b53c186e28ce83962639151a63c50be3fdddad2348c7a634bdfd85d8"
+
+  url "https://github.com/macstadium/orka-desktop/releases/download/#{version}/OrkaDesktop-#{version}.dmg"
+  name "Orka Desktop (Beta)"
+  desc "Run macOS virtual machines locally and build images for use with Orka"
+  homepage "https://github.com/macstadium/orka-desktop"
+
+  livecheck do
+    url :url
+    regex(/v?(\d+(?:\.\d+)+-beta\.\d+)/i)
```
Comment (bevanjkay) [inline Casks/o/orka-desktop.rb:12]:
```suggestion
    regex(/v?(\d+(?:\.\d+)+(?:-beta\.\d+)?)/i)
```

This should continue to work once releases are stable, however we can also likely remove the `regex` at that time.

Comment (cameronbroe) [reply Casks/o/orka-desktop.rb]: Thanks for the suggestion! I went on and updated the regex

**Review by p-linnane — APPROVED** [review]

**Pull request merged**